### DISCUSSION TITLE
Just confused

### DISCUSSION THREAD

**Discussion by AdenJustice** [oUF-wow/oUF#701] [Q&A] [answered]
https://www.townlong-yak.com/framexml/live/Blizzard_FrameXMLBase/Constants.lua#409

max totems are 4, but in the example of `totems.lua`, it creates 5 totem objects. Is there any specific reason?

**Comment by p3lim** [comment, marked as answer]
Probably a typo in the example, I'll get that fixed.

**Comment by p3lim** [comment]
Seems like GitHub doesn't cross-reference in discussions.
Opened a PR with a fix: #702
